### ISSUE TITLE
add rocketchat event handler

### ISSUE THREAD

**Issue by alekibango** [influxdata/kapacitor#1857]
please add rocketchat handler for events, making chat posts to user/channel

https://rocket.chat/docs/developer-guides/rest-api/chat/postmessage/

**Comment by llmuculusll** [comment]
+1

**Comment by alekibango** [comment]
i found out you can use slack handler  for rocketchat too.  but having rocketchat handler would be better.

**Comment by omerkarabacak** [comment]
+1

**Comment by dokime7** [comment]
+1

**Comment by joseluisq** [comment]
Are some there news about it?

**Comment by Si-Richards** [comment]
+1

**Comment by Si-Richards** [comment]
For anyone looking for a Rocketchat handler, just use the Slack handler.
The integration works the same and ideally, influx would just clone the handler and rename it Rocketchat for posterity!

**Comment by joseluisq** [comment]
> For anyone looking for a Rocketchat handler, just use the Slack handler.

Yeah, we ended up using the Slack handler too at the time and it worked.
